src/components/Book.js

```diff
@@ -29,7 +29,7 @@ const Book = (props) => {
 
 Book.propTypes = {
   bookItem: PropTypes.exact({
-    id: PropTypes.number,
+    item_id: PropTypes.number,
```

Comment:
- [ ] try changing this proptype to string as it currently throws an error in the console